test/unit/email_rate_limit_test.rb

```diff
@@ -13,9 +9,9 @@ def seed_array
     invalid_values        = last_clicked_days_ago.to_a - valid_values
     day_ago               = rand(last_clicked_days_ago)
     clicked_ago           = valid_values.sample
-    assert EmailRateLimit.new(day_ago).now?(clicked_ago), "Expected  EmailRateLimit.new(#{day_ago}).now?(#{clicked_ago}) to be true, was not"
+    assert_true { EmailRateLimit.new(day_ago).now?(clicked_ago) }
```

Comment:
Will this give the numbers in the output? Or will it always show the same string "EmailRateLimit.new(day_ago).now?(clicked_ago)". 

That's why I added those outputs, because otherwise when the tests randomly fail it's impossible to know why.